db-engine/src/main/java/com/splicemachine/db/impl/sql/compile/ScanCostFunction.java

```diff
@@ -67,7 +67,8 @@
 public class ScanCostFunction{
     public static final Logger LOG = Logger.getLogger(ScanCostFunction.class);
 
-    private static final int OLTP_INDEXLOOKUP_LATENCY_FACTOR = 10;
+    private static final int OLTP_INDEXLOOKUP_COST_PER_ROW = 40;
+    private static final int OLAP_INDEXLOOKUP_COST_PER_ROW = 80;
```

Comment:
I'm not sure we want to used fixed per-row costs.  Shouldn't this be like local and remote costs which are scaled based on the row width?  For example: totalRowCount*totalSelectivity*remoteLatency*(1+**colSizeFactor**/1024d);

Reply:
Can we at least open a JIRA to tackle this at a later point?

Reply:
Yes, I'll make new commits to improve here. Jira: https://splicemachine.atlassian.net/browse/DB-11795